### PR TITLE
Feature/events

### LINKED ISSUE #12: Single Event Page "Event" Section

### PR DESCRIPTION
**Pull request by Akalanka47000**
Added events page with a search bar, filter options still left to implement
https://user-images.githubusercontent.com/73662613/124105670-156c7d80-da81-11eb-9c8e-f8437518c0ba.mp4

### CODE REVIEW

**Comment by sithumonline** [comment]
@Akalanka47000 please provide screenshots

**Comment by LayanJay** [comment]
This implementation is quite impressive. But the fact is it's not specified in the design document. If you like, let me know I can give you access to update the design document according to your implementation. 

And keep in mind before you start work on something, join the discussion, or assign yourself to the relevant issue(s). It will definitely save other members' time. 👍

File: components/Event.tsx [hunk 1]
```diff
@@ -0,0 +1,76 @@
+import {useEffect } from 'react'
+import Aos from '../node_modules/aos/dist/aos.js'
+import '../node_modules/aos/dist/aos.css'
+
```
Comment (sithumonline) [inline components/Event.tsx]: Rm gaps between imports

File: components/NavSearch.tsx [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+            onChange={handleParam()}
+          />
+          <button
+            type="submit"
+            className="m-3 mt-0  md:my-4 mx-0 md:mx-5 w-full md:w-1/4 h-10 md:h-auto text-lg font-semibold bg-gradientBlue text-white md:text-white  shadow-xl hover:bg-gradientPurple duration-150 transition ease-in font-medium hover:text-white "
+          >
+            Search
+          </button>
+        </form>
+
```
Comment (sithumonline) [inline components/NavSearch.tsx]: Rm the gaps

File: pages/events.tsx [hunk 3]
```diff
@@ -0,0 +1,107 @@
+
+
+// components
```
Comment (sithumonline) [inline pages/events.tsx]: Rm gaps between imports

**Review by sithumonline — CHANGES_REQUESTED** [review]
Rm gaps and unwanted comments

**Comment by sithumonline** [comment]
@Akalanka47000 resolve conflicts and as @LayanJay mentioned, please assign yourself before the implementation.

**Comment by chamodshehanka** [comment]
@sithumonline don't forget to use labels

**Comment by chamodshehanka** [comment]
> This implementation is quite impressive. But the fact is it's not specified in the design document. If you like, let me know I can give you access to update the design document according to your implementation.
> 
> And keep in mind before you start work on something, join the discussion, or assign yourself to the relevant issue(s). It will definitely save other members' time. 👍

@sithumonline as I mentioned earlier assign issues to members, and also members can send a Draft(Work in Progress - WIP) PR too.

**Comment by sithumonline** [comment]
@Akalanka47000 CI failed, Please remove `;` and remove unused `e`.

**Pull request closed without merging**